src/python/src/grpc/_adapter/_call.c

```diff
@@ -236,6 +250,9 @@ static PyMethodDef methods[] = {
     {"complete", (PyCFunction)pygrpc_call_complete, METH_O,
      "Complete writes to this call."},
     {"accept", (PyCFunction)pygrpc_call_accept, METH_VARARGS, "Accept an RPC."},
+    {"add_metadata", (PyCFunction)pygrpc_call_add_metadata, METH_VARARGS,
+     "Add metadata to the call. May not be called after invoke or "
+     "premetadata."},
```

Comment:
The "or premetadata" smells stale here, given that calling add_metadata after premetadata is how terminal metadata gets added, right?